source/install/desktop-app-changelog.rst

```diff
@@ -17,9 +17,11 @@ Release v5.2
 --------------
 
 - **v5.2.2, release day TBD**
- - Fixing an issue on ``.msi`` installers where the onboarding screen is displayed even when there is a preconfigured server list `MM-48079 <https://mattermost.atlassian.net/browse/MM-48079>`_.
- - Fixing an issue where a crash can occur when a download list includes corrupt data `MM-48483 <https://mattermost.atlassian.net/browse/MM-48483>`_.
- - Fixing an issue where ``AppImageLauncher`` still creates a bad shortcut that causes the app not to launch `MM-48557 <https://mattermost.atlassian.net/browse/MM-48557>`_.
+ - Added ARM64 build (beta) for Windows/Linux.
+ - Fixed an issue on ``.msi`` installers where the onboarding screen was displayed even when there was a preconfigured server list `MM-48079 <https://mattermost.atlassian.net/browse/MM-48079>`_.
```

Comment:
This does happen on EXE as well in some cases. We can just say Windows in general.